src/lib/components/index/hero.svelte

```diff
@@ -2,12 +2,12 @@
   import Link from '$lib/components/index/link.svelte';
 </script>
 
+<head>
+  <link rel="stylesheet" href="css/index.css" />
+</head>
+
 <div class="space-y-8">
-  <h1
-    class="mx-auto bg-gradient-to-r from-primary-100 via-accent-magenta to-accent-cyan bg-clip-text text-center text-3xl font-bold text-transparent dark:to-primary-400 md:text-6xl"
-  >
-    Good-first-issue-finder
-  </h1>
+  <h1 class="gradient-flow">Good-first-issue-finder</h1>
```

Comment:
Please don't play that animation, if the media query for reduced motion is set. In case the user is having reduced-motion, please display the gradients the same way they are currently, without animation 👍 

@EmmaDawsonDev (sorry for that unexpected ping 😅) handling the animation like this is probably the correct way I hope 🙏

Reply:
Sure, I will turn the animation off for the reduced motion, otherwise it will have the animation.